test/node_api/importer_test.dart

```diff
@@ -112,6 +112,18 @@ void main() {
               importer: allowInterop(expectAsync2((_, __) {}, count: 0)))),
           equalsIgnoringWhitespace('a { b: c; }'));
     });
+
+    test("attempts directory index is #4", () async {
+      var subDir = p.join(sandbox, 'dir');
+      await createDirectory(subDir);
+      await writeTextFile(p.join(subDir, 'index.scss'), 'a { b: c; }');
+      expect(
+          renderSync(new RenderOptions(
+              data: '@import "dir"',
+              includePaths: [sandbox],
+              importer: allowInterop(expectAsync2((_, __) {}, count: 0)))),
+          equalsIgnoringWhitespace('a { b: c; }'));
+    });
```

Comment:
Index file support is more a part of the language than the API; it's an aspect of what it means for a filesystem import will mean. Which is to say, I think the sass-spec tests are sufficient and we don't need implementation-specific tests like this.